test/e2e-common.sh

```diff
@@ -392,14 +392,20 @@ function install() {
     echo "Patch config-network to enable encryption features"
     toggle_feature system-internal-tls enabled config-network
 
-    # This is currently only supported by kourier
     if [[ "$INGRESS_CLASS" == "kourier.ingress.networking.knative.dev" ]]; then
       toggle_feature cluster-local-domain-tls enabled config-network
     fi
 
+    if [[ "$INGRESS_CLASS" == "istio.ingress.networking.knative.dev" ]]; then
```

Comment:
combine with the if above, to not miss it while reading?

Reply:
fixed